apps/worker/packages/generic/src/index.ts

```diff
@@ -1,3 +1,4 @@
+import hostProcess from 'process'
```

Comment:
Should this be `import * as hostProcess from 'process'` as it used in `apps/appcontainer-node/packages/generic/src/appContainer.ts`?